fcd_sale_order_rpt_invoice/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_fcd_sale_report_invoice_user,fcd.sale.report.invoice user,fcd_sale_order_rpt_invoice.model_fcd_sale_report_invoice,sales_team.group_sale_salesman,1,0,0,0
```

Comment:
Why has this access been downgraded to Salesman? Sales Manager should be the right one. And remove group access configuration for the related menu, because becomes unnecessary

Reply:
The manager was correct but he had seen that a user also has read permissions in account.move.line, anyway I change it now as it was before